src/components/picture-in-picture/picture-in-picture.tsx

```diff
@@ -56,6 +57,39 @@ class PictureInPicture extends Component<any, any> {
     }
   ];
 
+  /**
+   * Creates an instance of PictureInPicture.
+   * @memberof PictureInPicture
+   */
+  constructor(props: any) {
+    super(props);
+    registerToBottomBar(COMPONENT_NAME, props.player, () => this.registerComponent());
+  }
+
+  registerComponent(): any {
+    return {
+      ariaLabel: () => this.getComponentText(),
+      displayName: COMPONENT_NAME,
+      order: 5,
+      svgIcon: () => this.getSvgIcon(),
+      onClick: () => this.togglePip(),
+      component: () => {
+        return getComp({...this.props, classNames: [style.upperBarIcon]});
```

Comment:
i dont like how all the classes have the same getComp function, but on the other hand if i remember right we can't put it on an interface.

does the syntax 

return PictureInPicture({...this.props, classNames: [style.upperBarIcon]}) work here ?
if not, i'd prefer to somehow have reuse instead of again, having very similar functions
it's also possible to have an entirely separate class which will only have a function which returns <PictureInPicture>, and then have one such class per every component, but while more organized it seems a bit overkill to me to have it for one single method...

Reply:
update- as discussed, leaving it the way it is now (just refactoring the name to `getComponent`)